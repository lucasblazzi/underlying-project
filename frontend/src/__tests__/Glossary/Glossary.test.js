import React from 'react';
import Glossary from '../../pages/Glossary/index';
import renderer from 'react-test-renderer'
import {Container} from "reactstrap"

it('PÃ¡gina', () => {
    const tree = renderer.create(<div class = "page-content"/>).toJSON();
    expect(tree).toMatchSnapshot();
})

it('Container', () => {
    const tree = renderer.create(<Container/>).toJSON();
    expect(tree).toMatchSnapshot();
})