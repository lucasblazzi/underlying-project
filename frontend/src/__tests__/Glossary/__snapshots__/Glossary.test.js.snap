// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container 1`] = `
<div
  className="container"
/>
`;

exports[`PÃ¡gina 1`] = `
<div
  class="page-content"
/>
`;
